source/less/blocks/example.less

```
@@ -0,0 +1,100 @@
+.example__wrapper{
+  .center;
+  background-color: @bg_example;
+}
+.example__title{
+  .font__size(36px, 40px);
+  color: @title_color;
+  margin: 0;
+  padding-top: 27px;
+}
+.example__subtitle{
+  .font__arial;
+  .font__size(14px, 18px);
+  margin-top: 33px;
+  color: @special_dark-grey;
+}
+.total__list{
+  .li-none;
+  display: flex;
+  justify-content: space-between;
+}
+.total__item{
+  position: relative;
+}
+.total__title{
+  .size (124px, 56px);
+  .font__size(24px, 24px);
+  color: @title_color;
+  display: block;
+  text-transform: uppercase;
+  font-weight: 400;
+  padding-top: 14px;
+  border: 1px solid @special_grey;
+  border-radius: 5%;
+  text-align: center;
+  margin-bottom: 0;
+}
+.total__subtitle{
+  .font__arial;
+  .font__size(12px, 12px);
+  color: @special_dark-grey;
+  position: absolute;
+  margin: 0;
+  top: 66px;
+  right: 20px;
+  background: @bg_example;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+.subtitle__block{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.maney-total{
```

```suggestion
.money-total{
```
Заодно возникает вопрос - а это БЭМ блок?